src/pretix/api/serializers/event.py

```diff
@@ -0,0 +1,11 @@
+from rest_framework import serializers
+
+from pretix.base.models import Event
+
+
+class EventSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Event
+        fields = ('name', 'slug', 'live', 'currency', 'date_from',
```

Comment:
Yes, it's intentional. As I'm currently working mostly offline, what you see here is not my latest state. In the current version, you currently can *only* get events as a nested resource, e.g. ``/organizers/<slug>/events/`` and ``/organizers/<slug>/events/<slug>/items/<id>/``. There are some very convincing reasons for this (i.e. its what you want most of the time and the required lookups to enforce permissions would be very compley otherwise), but there are of course some downsides as well. This is one of quite a few API design questions I wanted to talk to you about after the long weekend ;)